src/schema/definition.rs

```diff
@@ -27,9 +27,9 @@ impl Definition {
   }
 
   /// Gets type of a certain name.
-  pub fn find_type<K>(&self, tmp_name: K) -> Option<&Type> where K: Into<Key> {
-    let name = tmp_name.into();
-    self.types.iter().find(|type_| type_.name() == name)
+  // TODO: Should `N` really be borrowed?
```

Comment:
Oh, sorry I didn't know you were answering the comment :blush: 

I thought moving was the canonical way of doing this, [after some research](http://doc.rust-lang.org/std/collections/struct.BTreeMap.html#method.get), I guess not. I'll remove the `TODO`.